test/mpi/maint/jenkins/set-xfail.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env zsh
```

Comment:
Does this really require `zsh`? Most of the time it doesn't matter, but sometimes `zsh` isn't available by default.

Reply:
Bash is fine.